nntrainer/tensor/cache_loader.cpp

```diff
@@ -195,25 +195,30 @@ int CacheLoader::cancelAsync(int id) {
 }
 
 unsigned int CacheLoader::inActive(unsigned int order) {
-  std::set<unsigned int> exec_id = pool->getExecIDs(order);
-  for (auto &id : exec_id) {
-    std::shared_ptr<CacheElem> elem = pool->getCacheElem(id);
-    std::list<std::shared_ptr<CacheElem>> actives = pool->getActiveElems();
-    int load_task_id = elem->getLoadTaskID();
-    if (load_task_id >= 0) {
-      load_task_executor->releaseTask(load_task_id);
-      elem->setLoadTaskID(-1);
-      states[id] = LoadState::Unloading;
+  auto ids = pool->getExecIDsAll();
+  std::list<std::shared_ptr<CacheElem>> actives = pool->getActiveElems();
+  actives.clear();
+
+  for (auto id : ids) {
+    auto exec_id = id.second;
+    for (auto &id : exec_id) {
+      std::shared_ptr<CacheElem> elem = pool->getCacheElem(id);
+      int load_task_id = elem->getLoadTaskID();
+      if (load_task_id >= 0) {
+        load_task_executor->releaseTask(load_task_id);
+        elem->setLoadTaskID(-1);
+        states[id] = LoadState::Unloading;
+      }
```

Comment:
In the current code, the variable name "id" is duplicated.
How about changing the variable names?

```suggestion
  for (auto id : ids) {
    auto exec_ids = id.second;
    for (auto &exec_id: exec_ids) {
      std::shared_ptr<CacheElem> elem = pool->getCacheElem(id);
      int load_task_id = elem->getLoadTaskID();
      if (load_task_id >= 0) {
        load_task_executor->releaseTask(load_task_id);
        elem->setLoadTaskID(-1);
        states[exec_id] = LoadState::Unloading;
      }
```

Reply:
your right !!! thank you for review